src/components/Notes/Notes.tsx

```diff
@@ -0,0 +1,96 @@
+import { formatDateSince } from "../../utils/dateUtils";
+import { HorizontalDivider } from "../HorizontalDivider/HorizontalDivider";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { useNavigate } from "react-router-dom";
+import { Avatar, H1, H2, Stack } from "@deskpro/deskpro-ui";
+import styled from "styled-components";
+import { INote } from "../../api/types";
+import { useQueryWithClient } from "@deskpro/app-sdk";
+import { getUsersByIds } from "../../api/api";
+import { useMemo } from "react";
+
+type Props = {
+  notes: INote[];
+  id: string;
+};
+
+const HTMLDiv = styled.div`
+  & > p {
```

Comment:
use this for normalize styles https://github.com/DeskproApps/basecamp/blob/master/src/styles.ts

example: https://github.com/DeskproApps/basecamp/blob/master/src/components/common/Typography/DPNormalize.tsx#L14

Reply:
I only want the margin to be 0, don't think there's need to apply a few properties when only one is needed

Reply:
These styles are written because of the pain I have faced))) Try to check all the cases described in the styles and you'll see